PowerPointLabs/PowerPointLabs/Models/PowerPointSlide.cs

```diff
@@ -1241,5 +1241,10 @@ public void MakeShapeNamesUnique(Func<Shape, bool> restrictTo = null)
                 currentNames.Add(shape.Name);
             }
         }
+
+        public void DeleteSlideNumberShapes()
+        {
+            DeleteShapesWithPrefix("Slide Number Placeholder");

```

Comment:
This may not work for PowerPoint of other languages.
E.g. for Korean version, the shape name becomes something else and so can not be deleted by this prefix.